### PR TITLE
202 create a component which hides/shows divs

### PR DESCRIPTION
**Pull request by julianhahn**
closes #260

### CODE REVIEW

**Review by anbraten — APPROVED** [review]
👍🏾

File: packages/app/src/components/uiBlocks/GroupDropDownItem.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    },
+  },
+
+  setup() {
+    let itemIndex = ref<number | null>(null);
+    const itemCount = inject<Ref<number>>('itemCount');
+    const activeItemIndex = inject<Ref<number | null>>('activeItemIndex');
+
+    if (!activeItemIndex || !itemCount) {
+      throw new Error('du bist schlecht und musst sauber programmieren!');
```
Comment (tcoro) [inline packages/app/src/components/uiBlocks/GroupDropDownItem.vue:52]: maybe throw a more appropriate error message? 🤭

**Review by tcoro — APPROVED** [review]
besides that error message looks 🇳🇮 🇨🇫

File: packages/app/src/components/uiBlocks/GroupDropDownItem.vue (hunk 1 above)
Comment (JanEberlein) [reply packages/app/src/components/uiBlocks/GroupDropDownItem.vue:52]: 😆

File: packages/app/src/components/uiBlocks/GroupDropDownItem.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
```
Comment (JanEberlein) [inline packages/app/src/components/uiBlocks/GroupDropDownItem.vue:52]:
```suggestion
      throw new Error('reference injection failed');
```

**Review by JanEberlein — APPROVED** [review]

**Pull request merged**